client/my-sites/earn/memberships/index.jsx

```diff
@@ -529,48 +534,120 @@ class MembershipsSection extends Component {
 
 	renderOnboarding( cta ) {
 		const isPayments = isEnabled( 'earn/rename-payment-blocks' );
+		const { translate } = this.props;
+
+		if ( isPayments ) {
+			return (
+				<Card>
+					<div className="memberships__onboarding-wrapper">
+						<div className="memberships__onboarding-column-info">
+							<h2 className="memberships__onboarding-header">
+								{ preventWidows( translate( 'Accept payments on your website.' ) ) }
+							</h2>
+							<p className="memberships__onboarding-paragraph">
+								{ preventWidows(
+									translate(
+										'WordPress.com Payments makes it easy to sell physical and digital goods, accept donations, charge for in-person services, build subscription newsletters, and more.'
+									)
+								) }
+							</p>
+							<p className="memberships__onboarding-paragraph">
+								{ preventWidows(
+									translate(
+										'One-time, monthly, and yearly credit and debit card payment options are supported. {{link}}Learn more about payments.{{/link}}',
+										{
+											components: {
+												link: <a href="/support/recurring-payments-button/" />,
```

Comment:
Incorrect support link, it's https://wordpress.com/support/wordpress-editor/blocks/payments/#related-fees